### PR TITLE
docs: changing wallet decimal_places

### PR DESCRIPTION
**Pull request by justinkekeocha**
documented how to override wallet decimal_places

### CODE REVIEW

File: docs/configuration.md [hunk 1]
```diff
@@ -40,4 +40,18 @@ class MyWallet extends WalletBase {
    echo $user->wallet->helloWorld();
 ```
 This same method above, can be used to extend the base `Transfer` and `Transaction` models and registering the extended models in the configuration file.
+### Changing wallet decimal places
+
+You can change the default wallet decimal places, from your extended wallet model by defining a `decimal_places` attribute on the model. This is useful when working with fractional numbers
+
```
Comment (rez1dent3) [inline docs/configuration.md:46]: Hello. I highly do not recommend this method. Better this way:

config/wallet.php
```php
    /**
     * Base model 'wallet'.
     */
    'wallet' => [
        ....
        'creating' => [
            'decimal_places' => 18,
        ],
       ....
    ],
```

Comment (justinkekeocha) [reply docs/configuration.md:46]: Okay. It will be nice to document what the creating array actually does and the possible values.

Comment (justinkekeocha) [reply docs/configuration.md:46]: Maybe you can edit the pull request

Comment (rez1dent3) [reply docs/configuration.md:46]: Yes, maybe, but I can’t give you any advice on the timing. If you do this earlier, it will be great.

Comment (justinkekeocha) [reply docs/configuration.md:46]: effected the necessary changes

**Pull request merged**